content/announcements/open-letter-meta-dma/technical-briefing.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "Detailed technical briefing: The Case for XMPP – Why Meta Must Embrace True Messaging Interoperability"
+layout: single_open_letter_meta_dma
+---
+
+## Introduction
+
+The European **[Digital Markets Act (DMA)](https://digital-markets-act.ec.europa.eu/about-dma_en)** was introduced to ensure fair competition and prevent large technology companies from exploiting their dominant positions. A key aspect of the DMA is the requirement for **Number-Independent Interpersonal Communication Services (NIICS)**—which includes messaging platforms like WhatsApp and Messenger—to provide true interoperability with third-party services. This requirement is essential to breaking down walled gardens, promoting user choice, and fostering a more competitive digital ecosystem.
+
+The **[XMPP Standards Foundation](https://xmpp.org/about/xmpp-standards-foundation/)** has been closely following the implementation of the Digital Markets Act (DMA) in the European Union, with a specific focus on **messaging interoperability**. As an organization dedicated to open messaging standards, we are committed to helping stakeholders navigate the principles and technical realities of interoperable messaging.
+
+For over 25 years, we have developed and refined the **eXtensible Messaging and Presence Protocol (XMPP)**—a robust, extensible, and open standard that has fostered messaging interoperability worldwide. The XSF, through its membership requirements, and XMPP as a whole, by way of the distributed extensibility of the protocol, are independent of any vendor. Some of our members have been involved since XMPP’s inception in 1999, and several have already provided guidance to the European Union, serving as neutral experts to highlight the extensive resources and expertise our Foundation has cultivated.
+
+However, we now feel compelled to raise our voice more strongly. The direction taken by major gatekeepers of proprietary messaging networks raises serious concerns. Their proposed interoperability solutions appear, at best, as half-measures designed to maintain the status quo and, at worst, as deliberate strategies to reinforce their market dominance under the guise of compliance.
+
+Meta, designated as a [gatekeeper under the DMA](https://digital-markets-act.ec.europa.eu/gatekeepers_en), is the only company in the messaging sector subject to these new interoperability requirements, given its control over WhatsApp and Messenger. Yet, Meta’s current DMA proposal falls short of the European Union’s objectives. Instead of fostering true interoperability, it risks further entrenching Meta’s dominance—strengthening its hold on user data and reinforcing network effects that stifle competition.
```

Comment:
[...] falls short of the European Union's objectives, and offers no clear path to the future requirements. Instead of fostering true interoperability, it risks being seen as [...]